examples/mobile/HomeApp/breakfast-menu.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, user-scalable=no">
+	<title>Order your breakfast</title>
+	<link href="lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link rel="stylesheet" href="css/style.css" />
+	<script data-build-remove="false" src="lib/tau/mobile/js/tau.js"></script>
+</head>
+<body>
+	<div class="ui-page">
+		<header>
+			<h1>Choose your breakfast</h1>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-divider ui-li-has-checkbox">
+					<input name="european" type="checkbox" />
+					<div class="ui-li-text">European style</div>
+				</li>
+				<li class="ui-li-divider ui-li-has-checkbox">
+					<input name="asian" type="checkbox" />
+					<div class="ui-li-text">Asian Style</div>
+					
+				</li>
+				<li class="ui-li-divider ui-li-has-checkbox">
+					<input name="us" type="checkbox" />
+					<div class="ui-li-text">US Style</div>
+				</li>
+				<li class="ui-li-divider ui-li-has-checkbox">
+					<input name="vege" type="checkbox" />
+					<div class="ui-li-text">Vegetarian</div>
+				</li>
+				<li class="ui-li-divider ui-li-has-checkbox">
+					<input name="vegan" type="checkbox" />
+					<div class="ui-li-text">Vegan</div>
+				</li>
+				<li class="ui-li-divider ui-li-has-checkbox">
+					<input name="keto" type="checkbox" />
+					<div class="ui-li-text">Paleo/Keto</div>
+				</li>
+				<li class="ui-li-divider ui-li-has-checkbox">
+					<input name="lactose" type="checkbox" />
+					<div class="ui-li-text">Lactose Free</div>
+				</li>
+				<li class="ui-li-divider ui-li-has-checkbox">
+					<input name="gluten" type="checkbox" />
+					<div class="ui-li-text">Gluten Free</div>
+				</li>
+			</ul>
+		</div>
+		<footer class="ui-bottom-button">
+			<a href="#" class="ui-btn ui-btn-flat">Submit your order</a>
+		</footer>
```

Comment:
Submit order could go back to home app

Reply:
to `#main` page

Reply:
Now it's ok.
But if you "Submit your order", would you go  the payment page or payment site, please?
It's just enough by image.